consensus/bitcoin-ng.md

```diff
@@ -1,22 +1,13 @@
 Bitcoin-NG for Aeternity
 ==========
 
-Bitcoin-NG is a new consensus algorithm [1]. It depends on Proof-of-Work and uses chain of blocks with the most work done to resolve conflicts. Aeternity is PoW blockchain. Using Bitcoin-NG is very suitable for reducing on-chain confirmation latency. Bitcoin-NG is based on temporary leader that is elected randomly using Proof-of-Work.
-
-
-How to understand Bitcoin-NG?
-===
-
-To understand NG really quickly, we need to re-cap how we can think about classic Nakamoto consensus.
-
-Nakamoto consensus is based on crypto-puzzle that chooses a leader (miner) who confirms all the events included in the past block. New events are aggregated by all miners until one of them confirms them.
-
-Bitcoin-NG consensus is based on crypto-puzzle that chooses a leader (miner) who confirms all the events that will happen until the next miner solves the crypto-puzzle. The next miner takes over leadership.
+Bitcoin-NG is an evolution of Bitcoin protocol [1]. It uses the chain of blocks with the most work done to establish consensus. Consensus is guarded with Proof of Work. Using Bitcoin-NG improves latency of achieving on-chain consensus. Bitcoin-NG is based on temporary leader that is elected randomly using Proof-of-Work.
```

Comment:
Bitcoin-NG is an evolution of the Bitcoin protocol [1]. It still uses Nakamoto consensus but separates leader election and block production. This separation is done by introducing two kinds of blocks, key and micro. Key blocks are used for leader election. They don't contain any transactions but do require a solution to a Proof of Work puzzle. Once a leader has been found, they can produce multiple micro blocks. Micro blocks include transactions but do not require solutions to Proof of Work puzzles. Instead they need to be cryptographically signed be the current leader to be considered valid.

Splitting up leader election and block production allows a leader to produce micro blocks in rapid succession, increasing the transaction throughput significantly.